lib/index.js

```diff
@@ -85,7 +85,7 @@ internals.implementation = function (server, options) {
           // Authenticated
 
           return reply.continue({ credentials: credentials });
-        });
+        }, request);
```

Comment:
@ShujaatTariq does `reply.continue()` accept more than one argument? I can't find it in the API docs... http://hapijs.com/api#reply-interface http://hapijs.com/tutorials/auth
thanks!